pom.xml

```diff
@@ -145,8 +145,8 @@
         <cs.jaxb.version>2.3.0</cs.jaxb.version>
         <cs.jaxws.version>2.3.2-1</cs.jaxws.version>
         <cs.jersey-client.version>2.26</cs.jersey-client.version>
-        <cs.jetty.version>9.4.36.v20210114</cs.jetty.version>
-        <cs.jetty-maven-plugin.version>9.4.27.v20200227</cs.jetty-maven-plugin.version>
+        <cs.jetty.version>9.4.44.v20210927</cs.jetty.version>
+        <cs.jetty-maven-plugin.version>9.4.44.v20210927</cs.jetty-maven-plugin.version>
```

Comment:
I think the mvn plugin is for developers only, there are some issues because of which we still use 9.4.27... cc @sureshanaparti